Crashlytics/Crashlytics/Private/FIRStackFrame_Private.h

```diff
@@ -20,10 +20,10 @@ NS_ASSUME_NONNULL_BEGIN
 
 /**
  * This class is used in conjunction with recordExceptionModel to record information about
- * non-ObjC/C++ exceptions. All information included here will be displayed in the Crashlytics UI,
- * and can influence crash grouping. Be particularly careful with the use of the address property.
- *If set, Crashlytics will attempt symbolication and could overwrite other properities in the
- *process.
+ * non-ObjC/C++/Swift exceptions. All information included here will be displayed in the Crashlytics
```

Comment:
Why is Swift different from ObjC here?

Reply:
Do you mean why call it out separately from objective-c?

Reply:
Yes, why is ObjC special and different from the other languages?

Reply:
I think we're trying to call out non-objc, non-c++, non-swift exceptions - maybe we should write that out instead of using /